LICENSE

```diff
@@ -0,0 +1,692 @@
+License for Canonical Starter Pack
+==================================
+
+Unless otherwise stated, all code in this repository is licensed under the following GPL license.
```

Comment:
Silly nitpick: the whole file has hard breaks, so I'd suggest this line has it, too.
```suggestion
Unless otherwise stated, all code in this repository is licensed under
the following GPL license.
```